app/src/main/kotlin/com/wire/android/ui/conversation/model/Conversation.kt

```diff
@@ -0,0 +1,27 @@
+package com.wire.android.ui.conversation.model
+
+data class Conversation(
+    val userInfo: UserInfo,
+    val conversationInfo: ConversationInfo
+)
+
+data class ConversationInfo(
+    val name: String,
+    val memberShip: Membership = Membership.None,
```

Comment:
`membership` instead of `memberShip` ? 🚢

And as I mentioned in another comment, instead of checking for `membership != None`, we could just check if `membership != null` instead?

Reply:
for now I'll keep it to Membership.none, once we sync our tasks together we could go back to this topic